config.json.sample

```diff
@@ -0,0 +1,3 @@
+{
+    "featureManagerIsFullZoneProvisioningEnabled": false
+}
```

Comment:
If we only add those three to the sample we don't need to make the README more explicit.